source/mqtt5_client.c

```diff
@@ -2018,34 +2017,28 @@ PyObject *aws_py_mqtt5_client_get_stats(PyObject *self, PyObject *args) {
         return NULL;
     }
 
-    /* These must be DECREF'd when function ends */
+    /* These must be DECREF'd when function ends on error */
     PyObject *result = NULL;
 
     struct aws_mqtt5_client_operation_statistics stats;
     AWS_ZERO_STRUCT(stats);
 
     aws_mqtt5_client_get_stats(client->native, &stats);
 
-    result = PyObject_CallFunction(
-        get_stats_callback_fn_py,
-        "(KKKK)",
-        /* K */ (unsigned long long)stats.incomplete_operation_count,
-        /* K */ (unsigned long long)stats.incomplete_operation_size,
-        /* K */ (unsigned long long)stats.unacked_operation_count,
-        /* K */ (unsigned long long)stats.unacked_operation_size);
-    if (!result) {
-        PyErr_WriteUnraisable(PyErr_Occurred());
-        goto done;
-    }
-
+    result = PyTuple_New(4);
+    PyTuple_SET_ITEM(result, 0, PyLong_FromUnsignedLongLong((unsigned long long)stats.incomplete_operation_count));
```

Comment:
need to check results of PyTuple_New() and PyLongFromUnsignedLongLong() calls. Each may  report failure